api/const.go

```diff
@@ -2,9 +2,9 @@ package api
 
 // Request context key names
 const (
-	AppName string = "app_name"
-	Path    string = "path"
-	Call    string = "call"
+	App  string = "app"
```

Comment:
Why isn't this `AppID string = "app_id"`?

Reply:
Also, we may still need to keep "app_name" as an option here, since this is what middleware uses to optionally modify which app is being used.

Reply:
`App` can only be the app name, it's not an ID, I'll revert changes for this.